README.md

```diff
@@ -125,7 +137,8 @@ buf.as_slice().into_iter().map(|b| assert_eq!(b, 0xff)).for_each(drop);
 We think this can help a lot of database projects as well, where they typically have all the smarts in their database engine
 and they want the most simple (but fast) storage device. For a more elaborate example see some of the tests in mayastor/tests.
 
-To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local storage and remote storage using NVMe-oF TCP. Another advantage of the implementation is that if you were to remove
+To communicate with the children, the Nexus uses industry standard protocols. The Nexus supports direct access to local
```

Comment:
Why is there a newline break?

Reply:
Just to keep each line short in "code". Once rendered there's no line break